common/solver/batch_cg_kernels.hpp.inc

```diff
@@ -187,18 +187,16 @@ template <typename StopType, typename PrecType, typename LogType,
 __global__ void apply_kernel(
     const int max_iter, const gko::remove_complex<ValueType> tol,
     LogType logger, PrecType prec_shared, const BatchMatrixType a,
-    const gko::batch_dense::UniformBatch<const ValueType> left,
-    const gko::batch_dense::UniformBatch<const ValueType> right,
-    const gko::batch_dense::UniformBatch<ValueType> b,
+    const gko::batch_dense::UniformBatch<const ValueType> b,
```

Comment:
Looks like CG has not been specialized to 1RHS ? Does the multiple RHS version efficient enough for this ?

Reply:
No, that's just pending. We won't be using CG immediately, so it can be done later. Once I do CG and IDR as well, a bunch of cuda code for multiple RHS can also be removed.